packages/office-ui-fabric-react/src/components/MarqueeSelection/examples/MarqueeSelection.Basic.Example.tsx

```diff
@@ -48,73 +49,51 @@ const styles = mergeStyleSets({
       },
     },
   },
+  checkbox: {
+    margin: '10px 0',
+  },
 });
 
-const checkboxStyles: Partial<ICheckboxStyles> = { root: { margin: '10px 0' } };
-
-export interface IMarqueeSelectionBasicExampleState {
-  isMarqueeEnabled: boolean;
-}
-
-export class MarqueeSelectionBasicExample extends React.Component<{}, IMarqueeSelectionBasicExampleState> {
-  private _selection: Selection;
-  private _isMounted: boolean;
+const useForceUpdate = () => {
+  const [, setCount] = React.useState(0);
+  console.log('update');
+  return useConstCallback(() => setCount(current => current + 1));
+};
 
-  constructor(props: {}) {
-    super(props);
+export const MarqueeSelectionBasicExample: React.FunctionComponent = () => {
+  const [isMarqueeEnabled, { toggle: toggleIsMarqueeEnabled }] = useBoolean(true);
+  const forceUpdate = useForceUpdate();
 
-    this.state = {
-      isMarqueeEnabled: true,
-    };
-
-    this._selection = new Selection({
-      onSelectionChanged: () => {
-        if (this._isMounted) {
-          this.forceUpdate();
-        }
-      },
+  const selection = useConst(() => {
```

Comment:
Not very important but this could now be simplified to `const selection = useConst(() => new Selection(....))`